ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/project/ProjectServiceClient.java

```diff
@@ -160,6 +160,13 @@
   public Promise<Void> importProject(Path path, SourceStorageDto source) {
     String url = getBaseUrl() + IMPORT + encodePath(path);
 
+    if (url.contains("?")) {
+      url += "&clientId=" + appContext.getApplicationId().orElse("");
+    } else {
+
```

Comment:
Both parts of ```if``` should look the same.

Reply:
Improved by this [commit](https://github.com/eclipse/che/pull/6726/commits/013c55ec39634563b22afe00e1d5465de5c21aba)